packages/e2e-test-app-fabric/test/__snapshots__/ImageComponentTest.test.ts.snap

```diff
@@ -2387,7 +2387,7 @@ exports[`Image Tests An Image component can have a network callback 1`] = `
       "Sources": [
         {
           "Type": "Remote",
-          "Uri": "https://www.facebook.com/favicon.ico?r=1&t=<some_hash_here>",
+          "Uri": "https://www.facebook.com/assets/fb_lite_messaging/E2EE-settings@3x.png?r=1&t=1717025095331",
```

Comment:
FYI you'll have to update the logic here https://github.com/microsoft/react-native-windows/blob/b06795ef7c683825358c8150adb2107a816d98ac/packages/%40react-native-windows/automation-commands/src/dumpVisualTree.ts#L123 to be able to get consistent snapshots here.